### PR TITLE
Chore/page no index

### PR DESCRIPTION
**Pull request by rsdeus**
I have had to create an initializer to configuring the app rails to user the gem

### CODE REVIEW

File: config/initializers/deface.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```
Comment (froger) [inline config/initializers/deface.rb]: No need to have this file, let the main rails app handle deface. 
Deface is also present in extra_user_fields and bunch of other decidim gems.

File: Gemfile [hunk 2]
```diff
@@ -10,7 +10,7 @@ require_relative "#{base_path}lib/decidim/spam_signal/version"
 
 DECIDIM_VERSION = "0.24.3"
 
-gem "deface"
+gem "deface", '~> 1.8', '>= 1.8.1'
```
Comment (froger) [inline Gemfile]: Follow your gemspec here
```suggestion
gem "deface", '>= 1.8.1'
```

**Review by froger — CHANGES_REQUESTED** [review]
Hi,
You should take example on this module to implement deface: https://github.com/PopulateTools/decidim-module-extra_user_fields

See: 
- https://github.com/PopulateTools/decidim-module-extra_user_fields/blob/master/decidim-extra_user_fields.gemspec
- https://github.com/PopulateTools/decidim-module-extra_user_fields/blob/master/lib/decidim/extra_user_fields/engine.rb#L6
- https://github.com/PopulateTools/decidim-module-extra_user_fields/tree/master/app/overrides/decidim

**Comment by rsdeus** [comment]
Hi, all good now. Thanks for the help :heart:

File: Gemfile (hunk 2 above)
Comment (froger) [reply Gemfile]: Hi @rsdeus I see no change here ;)

**Review by froger — APPROVED** [review]

**Pull request merged**